specification/electronic-prescription-service-api.yaml

```diff
@@ -327,6 +331,64 @@ paths:
                 example:
                   value:
                     $ref: components/examples/example-a-validation-error-missing-field/Response-FhirError.json
+  /Task/$release:
+    post:
+      operationId: release
+      summary: Send a prescription release request to EPS
+      description: |
+        ## Overview
+        Use this endpoint to send a prescription release request to EPS.
+
+        This will gather a set of prescriptions based on the information passed to the endpoint: 
+        * to gather prescription information for an entire pharmacy, send owner information
```

Comment:
Do these make sense? Gather seems like they are getting but sending owner info seems like they are posting that they have downloaded them?